python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -378,8 +385,7 @@ def forward(self, x: torch.Tensor):
                 result = result.view(new_shape)
                 if self.bias is not None:
                     result += self.bias
-
-        return result.to(x.dtype)
+        return result
```

Comment:
This line of change causes this issue: https://github.com/analytics-zoo/nano/issues/639
@yangw1234 Please take a look.